src/react/databrowser/objects/ObjectListTable.jsx

```diff
@@ -35,11 +36,20 @@ type Props = {
 export default function ObjectListTable({ objects, bucketName, toggled, isVersioningType, prefixWithSlash }: Props){
     const dispatch = useDispatch();
     const nextMarker = useSelector((state: AppState) => state.s3.listObjectsResults.nextMarker);
-    const isItemLoaded = index => {
-        const shouldRefetch = nextMarker && index === objects.size - 1;
+    const loading = useSelector((state: AppState) => state.networkActivity.counter > 0);
+    const objectsLength = objects.size;
+    const isToggledFull = toggled.size > 0 && toggled.size === objectsLength;
+
+    const isItemLoaded = useCallback(index => {
```

Comment:
index parameter could be typed

Reply:
`flowtype/require-parameter-type` has not been enforced by our linter for now.